.github/workflows/duplicate_package_sample.yml

```diff
@@ -0,0 +1,24 @@
+name: Duplicate-Package-Sample
+
+on: push
+
+jobs:
+  duplicate:
+    name: duplicate package sample
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Run duplicate script file
+        run: bash ${GITHUB_WORKSPACE}/duplicate_package_sample.sh
+        
+      - name: Commit and Push
+        run: |
+          git config --global user.email "robel@chainsafe.io"
```

Comment:
Is it possible to add this to the pre-commit hook for example and now extra commit after?

Reply:
that could be better I have not tried, I'll check